App/Subapp01/views.py

```diff
@@ -494,14 +494,310 @@ def income_tax_calculator(request):
+    #     invest_80G = 25000 if invest_80d >= 25000 else invest_80d   
+    #     total_deduction = invest_80c+invest_80ccd+invest_80d+invest_80E+invest_80G
+
+    #     taxable_income = income_from_salary - total_exemption - total_deduction
+
+    #     if taxable_income <= 250000:  #2 Lakh 50 thousand
+    #         inc_tax = 0
+
+    #     elif taxable_income <= 500000: #5 Lakh
+    #         inc_tax = (taxable_income - 250000) * 0.05
+
+    #     elif taxable_income <= 750000: #7 lakh 50 thousand
+    #         inc_tax = (taxable_income - 500000) * 0.10 + 12500  
+
+    #     elif taxable_income <= 1000000: #10 Lakh
+    #         inc_tax = (taxable_income - 750000) * 0.15 + 37500 
+
+    #     elif taxable_income <= 1250000: #12 lakh 50 thousand
+    #         inc_tax = (taxable_income - 1000000) * 0.20 + 75000 
+
+    #     elif taxable_income <= 1500000: #15 lakh
+    #         inc_tax = (taxable_income - 1250000) * 0.25 + 125000 
+
+    #     else:
+    #         inc_tax = (taxable_income - 1500000) * 0.30 + 187500
+
+    #     return render(request, "app/income_33.html", {'gross_salary': gross_income,
+    #                                               'total_income': taxable_income,
+    #                                               'hra_exp': hra_exp,
+    #                                               'tax_ded': 1,
+    #                                               'hlt_ins_prm': 2,
+    #                                               'nps': 3,
+    #                                               'inc_tax': inc_tax,
+    #                                               'hlt_edu_cess': 4,
+    #                                               't': 4})
+
+        # invest_80c = 150000 if invest_80c >= 150000 else invest_80c
+        # invest_80ccd = 50000 if invest_80ccd >= 50000 else invest_80ccd
+        # invest_80d = 50000 if invest_80d >= 50000 else invest_80d
+
+        # if city == 'Mumbai' or city == 'Delhi' or city == 'Kolkata' or city == 'Bangalore':
+        #     hra = basic_pay*.5
+        # else:
+        #     hra = basic_pay*.4
+
+        # total_income = basic_pay - hra - \
+        # invest_80c - invest_80ccd - invest_80d
+        # if total_income <= 250000:  #2 Lakh 50 thousand
+        #     inc_tax = 0
+
+        # elif total_income <= 500000: #5 Lakh
+        #     inc_tax = (total_income - 250000) * 0.05
+
+        # elif total_income <= 750000: #7 lakh 50 thousand
+        #     inc_tax = (total_income - 500000) * 0.10 + 12500  
+
+        # elif total_income <= 1000000: #10 Lakh
+        #     inc_tax = (total_income - 750000) * 0.15 + 37500 
+
+        # elif total_income <= 1250000: #12 lakh 50 thousand
+        #     inc_tax = (total_income - 1000000) * 0.20 + 75000 
+
+        # elif total_income <= 1500000: #15 lakh
+        #     inc_tax = (total_income - 1250000) * 0.25 + 125000 
+
+        # else:
+        #     inc_tax = (total_income - 1500000) * 0.30 + 187500
+
+
+
+    # if age<=60:
```

Comment:
Why do we have such a big chunk of commented code? Is this required for some kind of book-keeping or reference? If so, please mention the reason of keeping this code piece in comments too.